akka-docs/src/main/paradox/cluster-usage.md

```diff
@@ -12,6 +12,47 @@ To use Akka Cluster, you must add the following dependency in your project:
+multiple smaller and more focused teams that can deliver new functionality more frequently and can
+respond quicker to business opportunities. Reactive Microservices should be isolated, autonomous, and have
+a single responsibility as identified by Jonas Bonér in the book
+[Reactive Microservices Architecture: Design Principles for Distributed Systems](https://info.lightbend.com/COLL-20XX-Reactive-Microservices-Architecture-RES-LP.html).
+
+In general we recommend against using Akka Cluster and actor messaging between _different_ services because that
+would result in a too tight code coupling between the services and difficulties deploying these independent of
+each other, which is one of the main reasons for using a microservices architecture.
+See the discussion on
+@scala[[Internal and External Communication](https://www.lagomframework.com/documentation/current/scala/InternalAndExternalCommunication.html)]
+@java[[Internal and External Communication](https://www.lagomframework.com/documentation/current/java/InternalAndExternalCommunication.html)]
+in the docs of the [Lagom Framework](https://www.lagomframework.com) (where each microservice is an Akka Cluster)
+for some background on this.
+
+Nodes of a single service (collectively called a cluster) require less decoupling. They share the same code and
+are managed together, as a set, by a single team or individual. For this reason, intra-service communication
+can take advantage of Akka Cluster and actor messaging, which is convenient to use and has great performance.
+
+Between different services [Akka HTTP](https://doc.akka.io/docs/akka-http/current) or
+[Akka gRPC](https://developer.lightbend.com/docs/akka-grpc/current/) can be used for synchronous (yet non-blocking)
+communication and [Akka Streams Kafka](https://developer.lightbend.com/docs/alpakka/current/) for asynchronous communication.
+[Alpakka](https://developer.lightbend.com/docs/alpakka/current/) provides many connectors for integration with
+external systems. All those communication mechanisms work well with streaming of messages with
+end-to-end back-pressure, and the synchronous communication tools can also be used for single request
+response interactions. It is also important to note that when using these tools both sides of the communication
+does not have to be implemented with Akka, or not even Java/Scala.
+
+We recommend the microservices architectural pattern and Akka Cluster should then be used inside a service for
+the communication between the nodes of that service.
+
+However, we acknowledge that microservices also introduce many new challenges and it's not the only way to
+build applications. A traditional distributed application may have less complexity and work well. For example
+for a small startup, with a single team, building an application where time to market is everything. Akka Cluster
+can efficiently be used for building such distributed application.
```

Comment:
Maybe we can say that we've seen multiple small companies start off like that because it enables quick iteration and devleopment before they had to split off things into many apps?

Reply:
(was just thinking of that japanese customer I talked to, they built it like that, got customers, great funding now, and ready to scale out more and separate concerns more and more)

Reply:
yes'll try to work in some more + here